backend/inspirehep/records/marshmallow/literature/base.py

```diff
@@ -49,6 +51,19 @@ class Meta:
     report_numbers = NonHiddenRaw(dump_only=True)
 
 
+class LiteratureApiSchema(LiteraturePublicSchema):
```

Comment:
ditto

Reply:
It's the same, I can't add it to PublicSchema because I'll change detailed page